scripts/release/common.sh

```diff
@@ -41,7 +41,7 @@ function __git_commit_subject() {
 
 # Returns the branch name of the current git repository
 function git_get_branch() {
-  git symbolic-ref --short HEAD
+  git symbolic-ref --short HEAD 2>/dev/null || git branch --remote --contains | cut -d "/" -f2
```

Comment:
Can you please add a comment explaining the two cases?

Reply:
Done1